src/Common.am

```diff
@@ -45,6 +45,7 @@ DEFS += -DDEFAULT_CONFIG_FILE=\"$(DEFAULT_CONFIG_FILE)\" \
 CLEANFILES =
 check_PROGRAMS =
 TESTS =
+DONOTTEST =
```

Comment:
I recommend avoiding negative names (e.g., those containing "no" and "not") because they often make code more difficult to read/interpret, especially in the presence of other logical "not" operations.

Adding underscores also makes these ALLCAPS names more readable IMO.

I do not insist on any specific spelling, but can suggest

* EXCLUDE_FROM_HDR_TEST
* EXCLUDE_FROM_HDR_TESTING
* HDR_TEST_EXCLUSION
* HDR_TEST_BAN

Reply:
using EXCLUDE_FROM_HDR_TESTING